testsuite/integration-arquillian/tests/other/console/src/main/resources/themes/qe/login/messages/messages_en.properties

```diff
@@ -0,0 +1,4 @@
+doAccept=Yes
```

Comment:
No harm. It's fine to include, but personally I would stick with purely on the termsText and have a separate test for custom themes that do bigger modifications.